generators/kubernetes-helm/templates/app/chart.yml.ejs

```diff
@@ -0,0 +1,25 @@
+name: <%= app.baseName.toLowerCase() %>
+version: <%= jhipsterVersion %>
+appVersion: 1.0.0
+<%_ if (composeApplicationType === 'microservice') { _%>
```

Comment:
I don't understand this part. First this line fails for monoliths as `composeApplicationType` is undefined for a monolith. I would simply write : 

```
description: <%= app.baseName.toLowerCase() %> is a JHipster based <%= app.applicationType %> application
```